src/sqlancer/citus/README.md

```diff
@@ -0,0 +1,141 @@
+# SQLancer for Citus (PostgreSQL extension)
```

Comment:
I think a better location for this would be the https://github.com/sqlancer/sqlancer/tree/master/docs directory (unless you have a strong opinion against this). We can also link to it from the main `README.md`.

There is much information that is redundant to the main `README.md`, which is undesirable, since it makes it difficult to keep the documentation is up-to-date. Please document only the Citus-specific parts.